src/allmydata/scripts/create_node.py

```diff
@@ -344,6 +348,74 @@ def create_node(config):
         os.mkdir(basedir)
     write_tac(basedir, "client")
 
+    # if we're doing magic-wormhole stuff, do it now
+    if config['join'] is not None:
+        print >>out, "Opening wormhole with code '{}'".format(config['join'])
+        relay_url = config.parent['wormhole-server']
+        print >>out, "Connecting to '{}'".format(relay_url)
+
+        wh = wormhole.create(
+            appid=u"tahoe-lafs.org/lafs",
```

Comment:
Did we release anything with the "v1" appid in it?

Also, is the appid the right place to do versioning like this, or in the capabilities exchanged as the first messages? I guess the "pro" of doing it in the appid is no possibilities of "crossing the streams" and accidentally receiving a grid-invite as a magic-folder-invite..? (but still, in such a case the capabilities wouldn't match anyway...)